test/quick.ts

```diff
@@ -0,0 +1,58 @@
+const seneca = require('seneca')
+// const Shared = require('seneca-store-test')
+
+const KnexStore = require('../src/knex-store')
+const DbConfig = require('./config/database/config')
+
+
+  const test = seneca()
+    .test()
+    .use('entity')
+    .use(KnexStore, { DbConfig })
+    .use('promisify')
+
+  console.log(test)
```

Comment:
Please do not leave `console.log`'s in automated tests.

Reply:
it's actually ok here - quick.js is meant to be a "quick" script to smoke check the lib - it's not part of the test suite